compiler/optimizer/LocalOpts.cpp

```diff
@@ -1166,6 +1166,7 @@ int32_t TR_BlockManipulator::performChecksAndTreesMovement(TR::Block *newBlock,
          ;
 
       TR::Block *nextBlock = prevBlock->getNextBlock();
+      // If this condition is satisfied, nextBlock != NULL
```

Comment:
what's the point of adding this comment? And how did you come to this conclusion?